src/SailthruChannel.php

```diff
@@ -67,6 +68,23 @@ public function send(
                 static::getDefaultVars()
             );
 
+            if (config('services.sailthru.whitelist.enabled') === true) {
+                if(!Str::is(
+                    config('services.sailthru.whitelist.domains'),
+                    $message->getToEmail()
+                )){
+                    Log::info(
+                        'Sailthru email not sent due to domain whitelist',
```

Comment:
```suggestion
                        'Sailthru email not sent to ' . $message->getToEmail() . ' due to domain whitelist limitations',

Reply:
Careful with exposing PII information.. you either obscure the email or not log it.

Reply:
@atimbang I dont particularly see a problem with showing the emails in our logs.The logs are not public, are they?